sshnp_register_tool/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: sshnp_register_tool
+description: A sample command-line application.
+version: 1.0.0
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.18.0 <3.0.0'
+
+# dependencies:
+#   path: ^1.8.0
+
+dev_dependencies:
+  lints: ^2.0.0
+  test: ^1.16.0
+dependencies:
```

Comment:
Please add a dependabot config to keep these up to date. There should also be tests to ensure that everything still works when a dependency is bumped, and when those tests pass we can build and release binaries.